### DISCUSSION TITLE
Feat. Request: Consider to add "Metronome" relaxing sounds …

### DISCUSSION THREAD

**Discussion by just4t** [CodeDead/opal#107] [Ideas]
First all all, thanks for this wonderfull app …
Please, consider to add "Metronome" relaxing sounds to the available sounds list.
Thanks in advance,
best
just4t

**Comment by CodeDead** [comment]
You can count on it. Currently waiting on JDK upgrade 21 from Adoptium to release before working on Opal again. I'll add your feature when the time comes :+1:

**Comment by CodeDead** [comment]
Done:
https://github.com/CodeDead/opal/pull/110
